src/app/pages/send/send-crypto-asset-form/form/brc-20/brc-20-choose-fee.tsx

```diff
@@ -45,6 +45,8 @@ export function BrcChooseFee() {
     amount: Number(amount),
     recipient,
   });
+  const recommendedFeeRate = feesList[1]?.feeRate.toString() || '';
```

Comment:
maybe a fee rate enum would work here for better readability

Reply:
you mean change feesList to object to get values by fee type? 
`feesList[BtcFeeType.Standart]` ?

Reply:
yeah

Reply:
seems, this will lead to quiet a huge refactor for the hotfix